registrace_bp.py

```diff
@@ -1,73 +1,80 @@
 from flask import Blueprint, render_template, request, redirect, url_for, flash, abort, current_app
-# from wtforms import Form, BooleanField, StringField, PasswordField, validators
+from wtforms import Form, BooleanField, StringField, PasswordField, IntegerField, validators
 from flask_login import current_user
 import db_funkce
 
 
+class RegistrationForm(Form):
+    username = StringField(
+        'Jméno*:', [validators.Required(message='Bez jména to nepůjde.')])
+    surname = StringField(
+        'Příjmení*:', [validators.Required(message='Bez příjmení to nepůjde.')])
+    street = StringField('Ulice,č.p./č.o.:')
+    city = StringField(
+        'Město/Obec*:', [validators.Required(message='Vyplň město, kde bydlíš.')])
+    postcode = IntegerField('PSČ*:', [validators.Required(message='Bez PSČ to nepůjde.'),
+                                      validators.Length(min=5, max=5, message='PSČ zadej bez mezer.')])
+    email = StringField('E-mail*:', [validators.Required(message='Bez mailu to nepůjde.', ),
+                                     validators.Email(message='Chybný tvar emailové adresy.'), validators.Length(min=4, max=40)])
+    phone = IntegerField('Telefon*:', [validators.Required(message='Bez telefonního čísla to nepůjde.'), validators.Length(
```

Comment:
U `IntegerField` nepůjdou zadat mezery, pokud chceš ten formát jako "XXX XXX XXX", tak to musíš udělat jako `StringField`, ale pozor, že to zase nepůjde uložit do databáze (upřímně bych v tomhle ale změnil tu databázi, aby se telefon ukládal jako String, ale to je na samostatný task a samostatnou změnu)....pro teď bych z toho ten Length validátor vyhodil.